create-doks-with-terraform-flux/README.md

```diff
@@ -1 +1,470 @@
+# Gitops stack using DOKS and Flux CD
```

Comment:
Can you make this title more "user goal" oriented?

Reply:
To be honest I don't know how we can change the title to be more "user goal" oriented. I mean it's already telling us what we're trying to accomplish in this tutorial. But maybe I'm missing the point here so any further explanation is welcome. Thanks.